src/components/ui/CheckboxUsageExample.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+/**
+ * This file is used as an example for the Checkbox components.
+ * Once we're done with the example, we can delete this file.
+ */
+import { ContainerIcon } from '@/assets/images/icons';
+import { Card, Checkbox, Icon, Label } from '@/components/ui';
+
+export const CheckboxUsageExample = () => {
+  return (
+    <div className="p-4 flex flex-col gap-4">
+      {/* Example with text */}
+      <Card asChild>
+        <div className="flex justify-between items-center py-2 px-4">
```

Comment:
I understand that you have made an update, but it's not what I intended. 

First, we should always avoid using the plain `<label>` tag everywhere, since we have defined the `<Label>` component. 

Secondly, what I originally meant was to substitute the previously specified `<div>` with a `<Label>` tag, which should have its `htmlFor` attribute referencing the `id` of the checkbox. Also, the existing `<Label>` tag needs to be replaced with a normal `<div>`.

Reply:
Addressed in this PR: https://github.com/genesis-tech-tribe/nishiki-frontend/pull/191